### DISCUSSION TITLE
many reconnect timeouts

### DISCUSSION THREAD

**Discussion by oshai** [redis/lettuce#2464] [Q&A]
We are using redis in gcp with the driver.

From a specific point in time our logs becomes full of:
```
2023-07-25 07:23:48,330 [lettuce-nioEventLoop-32-1 []] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [***:11008]: connection timed out: /***:11008
```

Any idea what can cause this?

**Comment by fr3d3rico** [comment]
Hi Ohad Shai! Did you find out what happened? I am experiencing this logs too. My app is running on Openshift cluster.

**Reply by oshai** [reply]
IIRC our issue was related to https://github.com/lettuce-io/lettuce-core/issues/339.
we set the following config:
```
useDynamicRefreshSources = false
enablePeriodicRefresh = true
enableAllAdaptiveRefreshTriggers
validateClusterNodeMembership(false)

```

**Reply by fr3d3rico** [reply]
Thank you for letting me know.

**Comment by mp911de** [comment]
Connection timeouts are mostly caused by either firewall rules or an IP address that isn't responding. In any case, it is an infrastructure issue.

**Reply by fr3d3rico** [reply]
Yeah! There is something happen in openshift... I appreciate your comment!
